tests/clone_test.go

```diff
@@ -840,3 +852,31 @@ var _ = Describe("[Serial]VirtualMachineClone Tests", Serial, func() {
 		})
 	})
 })
+
+func startCloneVM(virtClient kubecli.KubevirtClient, vm *virtv1.VirtualMachine) (*virtv1.VirtualMachine, error) {
+	patch, err := patch.New(patch.WithReplace("/spec/runStrategy", virtv1.RunStrategyAlways)).GeneratePayload()
+	if err != nil {
+		return nil, err
+	}
+
+	updatedVM, err := virtClient.VirtualMachine(vm.Namespace).Patch(context.Background(), vm.Name, types.JSONPatchType, patch, v1.PatchOptions{})
```

Comment:
- You could return directly here.
- You could avoid passing virt-client (as it is a singleton).

Reply:
Thanks. I honestly prefer passing the one from beforeeach instead of `virtClient := kubevirt.Client()`
but let me know if you insist on that one

Reply:
I thought we agreed passing all over the clientset is not useful and causes unneeded noise.
But I have no intention to insist on such nits given the context of this change.